pkg/client/patch.go

```diff
@@ -26,6 +26,9 @@ import (
 var (
 	// Apply uses server-side apply to patch the given object.
 	Apply = applyPatch{}
+
+	// Merge uses a raw merge strategy to patch the object.
```

Comment:
nit: this doc comment could be a little clearer -- e.g. `Merge uses the raw object as a merge patch, without modifications.  Use MergeFrom if you wish to compute a diff instead.`